### PR TITLE
Fix linechart active point on zoom mode

### PR DESCRIPTION
**Pull request by KaroMourad**
Fix: 
 - Brush `active` or `focused` point location on zoom
 
Add: 
 - `updateFocusedChart` function can call with an existing `active` or `focused` point(can call without receiving mouse position)

### CODE REVIEW

**Review by roubkar — APPROVED** [review]

File: aim/web/ui_v2/src/types/components/LineChart/LineChart.d.ts [hunk 1]
```diff
@@ -41,6 +47,6 @@ export interface IAttributesRef {
   updateScales?: (xScale: IGetAxisScale, yScale: IGetAxisScale) => void;
   setActiveLine?: (lineKey: string) => void;
   updateHoverAttributes?: (xValue: number) => void;
-  updateFocusedChart?: (mousePos: [number, number]) => IActivePoint;
+  updateFocusedChart?: (props?: IUpdateFocusedChartProps) => void;
```
Comment (arsengit) [inline aim/web/ui_v2/src/types/components/LineChart/LineChart.d.ts:50]: lets write function arguments as params instead of props

**Review by VkoHov — APPROVED** [review]

File: aim/web/ui_v2/src/types/components/LineChart/LineChart.d.ts (hunk 1 above)
Comment (KaroMourad) [reply aim/web/ui_v2/src/types/components/LineChart/LineChart.d.ts:50]: fine i will fix this , thanks

Comment (KaroMourad) [reply aim/web/ui_v2/src/types/components/LineChart/LineChart.d.ts]: for only simple functions we decide to do yes ? for react components we can use props as well

**Review by arsengit — APPROVED** [review]
great

**Pull request merged**